python-frontend/src/main/java/org/sonar/python/lexer/PythonLexer.java

```diff
@@ -41,6 +41,8 @@
   private static final String IMAGINARY_SUFFIX = "(j|J)";
   private static final String LONG_INTEGER_SUFFIX = "(l|L)";
   private static final String FORMATTED_STRING_PREFIX = "([fF][rR]?|[rR][fF]?)";
+  private static final String IDENTIFIER_START = "[\\p{Lu}\\p{Ll}\\p{Lt}\\p{Lm}\\p{Lo}\\p{Nl}_]";
+  private static final String IDENTIFIER_CONTINUE = "[" + IDENTIFIER_START + "\\p{Mn}\\p{Mc}\\p{Nd}\\p{Pc}]";
```

Comment:
The nesting of square brackets (IDENTIFIER_START contains square brackets) is a bit confusing.